### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by chliangGoogle**

### CODE REVIEW

**Review by maksymmalyhin — APPROVED** [review]

**Comment by maksymmalyhin** [comment]
Please fix CI before merging.

File: FirebaseMessaging/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# unreleased
-- [added] Added new API `FIRMessagingExtensionHelper exportDeliveryMetricsToBigQuery` that allows developers to enable notification delivery metrics to BigQuery. (#6181)
+# 2021-07 -- v8.5.0
+- [added] Added new API `FIRMessagingExtensionHelper exportDeliveryMetricsToBigQuery` that allows developers to enable notification delivery metrics that exports to BigQuery. (#6181)
```
Comment (paulb777) [inline FirebaseMessaging/CHANGELOG.md:2]: Perhaps more clear and concise, if it keeps the right meaning?

Added API `FIRMessagingExtensionHelper exportDeliveryMetricsToBigQuery` to enable notification delivery metric exports to BigQuery.

**Review by paulb777 — APPROVED** [review]
The CI failures (or flakes) look unrelated and could be dealt with in a separate PR.

**Comment by chliangGoogle** [comment]
I think the package.swift change is still needed so I will keep it in this PR?

**Pull request merged**